src/data/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface FavoritesData {
+  [i: string]: boolean;
```

Comment:
This index signature says that the `FavoritesData` object is allowed to have any string keys on its members, and their values are all `boolean`s.

In retrospect this type is only used in 2-3 files so a clearer organizational strategy might be to make a `hooks/favorites` folder that contains the favorites files and a `types.ts`. 🤷